### PR TITLE
[RW-8020][risk=no] Improve Updated Welcome email

### PR DESCRIPTION
**Pull request by NehaBroad**
Update Sign Code of Conduct on DAR page
Screen Shot 2022-03-24 at 12 51 38 PM

Email:

- Update Sign of Conduct case
-  Remove flex from bullet

image

---
**PR checklist**

- [ ] This PR meets the Acceptance Criteria in the JIRA story
- [ ] The JIRA story has been moved to Dev Review
- [ ] This PR includes appropriate unit tests
- [ ] I have run and tested this change locally
- [ ] I have run the E2E tests on ths change against my local UI and/or API server with `yarn test-local` or [yarn test-local-devup](https://github.com/all-of-us/workbench/blob/main/e2e/README.md#examples)
- [ ] If this includes a UI change, I have taken screen recordings or screenshots of the new behavior and notified the PO and UX designer
- [ ] If this includes an API change, I have updated the appropriate Swagger definitions and notified API consumers
- [ ] If this includes a new feature flag, I have created and linked new JIRA tickets to (a) turn on the feature flag and (b) remove it later

### CODE REVIEW

File: api/src/main/resources/emails/welcomeemail/content.html [hunk 1]
```diff
@@ -213,8 +213,8 @@
         </div>
         <div style="padding-bottom: 1.6rem">
           <div class="text bold-user-label" style="margin-bottom: 0.1rem">Step 3</div>
-          <div style="font-weight: 400; padding-bottom: 0.6rem; padding-top: 0.2rem">Sign the code
-            of conduct
+          <div style="font-weight: 400; padding-bottom: 0.6rem; padding-top: 0.2rem">Sign The Code
```
Comment (jmthibault79) [inline api/src/main/resources/emails/welcomeemail/content.html:216]: This doesn't look right to me... could you confirm with PO?

File: api/src/test/java/org/pmiops/workbench/api/ProfileControllerTest.java [hunk 2]
```diff
@@ -888,37 +889,83 @@ public void updateFamilyName_badRequest() {
 
   @Test
   public void resendWelcomeEmail_messagingException() throws MessagingException {
+    config.access.tiersVisibleToUsers =
```
Comment (jmthibault79) [inline api/src/test/java/org/pmiops/workbench/api/ProfileControllerTest.java:892]: please add a comment explaining why this is necessary

File: api/src/test/java/org/pmiops/workbench/api/ProfileControllerTest.java [hunk 3]
```diff
@@ -888,37 +889,83 @@ public void updateFamilyName_badRequest() {
...
     assertThat(response.getStatusCode()).isEqualTo(HttpStatus.NO_CONTENT);
     // called twice, once during account creation, once on resend
-    verify(mockMailService, times(2)).sendWelcomeEmail_deprecated(any(), any(), any());
+    verify(mockMailService, times(2))
+        .sendWelcomeEmail(any(), any(), any(), any(), anyBoolean(), anyBoolean());
     verify(mockDirectoryService, times(1)).resetUserPassword(anyString());
   }
 
+  @Test
+  public void sendUserInstructions_none_deprecated() throws MessagingException {
```
Comment (jmthibault79) [inline api/src/test/java/org/pmiops/workbench/api/ProfileControllerTest.java:937]: these new tests are not correct.  We already have the "deprecated" versions.  We need the undeprecated versions.

Comment (jmthibault79) [reply api/src/test/java/org/pmiops/workbench/api/ProfileControllerTest.java:937]: also need a "deleted instructions" undeprecated test

File: api/src/test/java/org/pmiops/workbench/api/ProfileControllerTest.java (hunk 2 above)
Comment (jmthibault79) [reply api/src/test/java/org/pmiops/workbench/api/ProfileControllerTest.java:892]: please also restore the deprecated versions of these tests

File: api/src/test/java/org/pmiops/workbench/mail/MailServiceImplTest.java [hunk 4]
```diff
@@ -101,6 +103,75 @@ public void testSendWelcomeEmail() throws MessagingException, ApiException {
...
+    assertThrows(
+        MessagingException.class,
+        () ->
+            service.sendWelcomeEmail(
+                CONTACT_EMAIL, PASSWORD, FULL_USER_NAME, INSTITUTION_NAME, true, true));
+    verify(mandrillApi, times(3)).send(any());
+  }
+
+  @Test
+  public void testSendWelcomeEmail_invalidEmail_RTAndCt() throws MessagingException {
```
Comment (jmthibault79) [inline api/src/test/java/org/pmiops/workbench/mail/MailServiceImplTest.java:129]: Please add local vars to make it clear what these 6 tests are doing.

Like: `boolean userHasRt = true`

File: api/src/main/resources/emails/welcomeemail/content.html [hunk 5]
```diff
@@ -252,50 +252,62 @@
...
-        <div class="text">
-          <a class="text-link" href="https://support.google.com/chrome/answer/95346">Chrome</a>
-          <span> is the primary browser we currently support for the Researcher Workbench.
+      <tr>
+        <td colspan="1">
+          <div class="bullet">
+            <img src="${BULLET_1}">
+          </div>
+        </td>
+        <td colspan="3">
```
[the remark this answers is not in the document]
Comment (evrii) [reply api/src/main/resources/emails/welcomeemail/content.html:261]: I updated the spacing and put a screenshot in the description. What do you think?

File: api/src/test/java/org/pmiops/workbench/api/ProfileControllerTest.java (hunk 2 above)
Comment (evrii) [reply api/src/test/java/org/pmiops/workbench/api/ProfileControllerTest.java:934]: This functionality will no longer be used as of the next production release, so we decided that the tests do not provide enough utility for the time needed to bring them back.

File: api/src/test/java/org/pmiops/workbench/api/ProfileControllerTest.java (hunk 3 above)
Comment (evrii) [reply api/src/test/java/org/pmiops/workbench/api/ProfileControllerTest.java:979]: Added a test to handle the un-deprecated case.

File: api/src/test/java/org/pmiops/workbench/mail/MailServiceImplTest.java (hunk 4 above)
Comment (evrii) [reply api/src/test/java/org/pmiops/workbench/mail/MailServiceImplTest.java:147]: Decided to not pursue this at this time. If we would like to pursue this convention in the future, we can discuss it.

**Pull request merged**